duckduckgo/macos-browser
# Polishing of popovers

Task/Issue URL: https://app.asana.com/0/1177771139624306/1200112778095263/f
CC: @samsymons 

**Description**:
PR includes two improvements:
1. Making fireproofing confirmation dialog shown even when user interacts with the website.
2. Matching the behaviour of fireproofing and bookmarking popovers, specifically:
- address bar button mouse over effect (turned on for fireproofing button)
- closing behaviour when user interacts with website (turned on for bookmarking popover)

**Steps to test this PR**:
1. Fireproof a website using the right options menu. Make sure the confirmation dialog - “XY is fireproofed” - is not hidden when you interact with website. It should be hidden automatically after 2.5 sec
2. Verify fireproofing and bookmarking popovers (displayed after clicking on appropriate buttons) have the same closing behaviour - they are closed when user interacts with other elements of the browser

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
